src/Kris/LaravelFormBuilder/FormHelper.php

```diff
@@ -72,9 +78,10 @@ class FormHelper
      * @param View    $view
      * @param array   $config
      */
-    public function __construct(View $view, array $config = [])
+    public function __construct(View $view, array $config = [], Translator $translator)
```

Comment:
Please move `$translator` to be 2nd param, and `$config` to be last, because it's optional.